probe-rs/src/architecture/arm/ap/generic_ap.rs

```diff
@@ -39,14 +39,14 @@ pub enum ApType {
     JtagComAp = 0x0,
     /// A AMBA based AHB3 AP (see E1.5).
     AmbaAhb3 = 0x1,
-    /// A AMBA based AHB2 and AHB3 AP (see E1.8).
-    AmbaAhb2Ahb3 = 0x2,
+    /// A AMBA based APB2 and APB3 AP (see E1.8).
```

Comment:
What are these changes? If they are "just" fixes to the names, should we land them separately?

Reply:
This specfic change yes.
It could be pushed independently but if you don’t mind, I’d like to get a proper/sensible commit chain before starting splitting them into separate PRs if deemed necessary for review purposes.

I’d rather avoid more painful rebases :S

Reply:
> I’d rather avoid more painful rebases :S

Yeah I guess +1700 line PRs don't help here :)